DBMS/Assignment1/library.sql

```diff
@@ -0,0 +1,128 @@
+/* created database*/
+CREATE DATABASE Library_Management;
+USE Library_Management;
+
+/* created table named members*/
+CREATE TABLE members(
+member_id INT PRIMARY KEY AUTO_INCREMENT,
+member_nm VARCHAR(20),
```

Comment:
ACTION: Avoid abbreviations, especially if they're just the type that removes vowels.